tests/test_config_flow.py

```
@@ -799,175 +726,55 @@ async def test_options_flow_init_selected_mode_shows_picker_step(
+            )
+            self.async_close = AsyncMock()
+
+        async def get_host_firmware_version(self) -> str:
+            """Return the fake firmware version."""
+            return self.firmware_version
+
+        async def get_system_info(self) -> dict[str, str]:
+            """Return static system metadata for validation."""
+            return {"name": "OPNsense"}
+
+        async def get_device_unique_id(self, expected_id: str | None = None) -> str:
```

Ruff (0.15.17)

[warning] Unused method argument: `expected_id`

(ARG002)